shell/store/growl.js

```diff
@@ -18,11 +18,14 @@ export const getters = {
 
 export const mutations = {
   add(state, data) {
-    state.stack.push({
-      id:      state.nextId++,
-      started: (new Date().getTime()),
-      ...data
-    });
+    state.stack = [
```

Comment:
Push was appending elements to the end of the state stack, while this is appending it to the beginning. is this change intentional?

Reply:
Definitely not intentional, good catch!